docs/docs/rnk-rpc-methods.md

```diff
@@ -0,0 +1,780 @@
+---
+title: RNK RPC Methods
+sidebar_position: 7
+description: Explore Reactive Network's RPC methods used for communication with nodes and ReactVMs.
+slug: /rnk-rpc-methods
+hide_title: true
+---
+
+## Overview
+
+This page provides an overview of Reactive Network's (RNK) RPC methods, which enable interaction with nodes and ReactVMs. The methods include transaction retrieval, log access, callback information, and more. Below is a detailed description of each method along with its parameters, usage examples, and responses.
```

Comment:
Well... to be precise here, it's an overview of certain unique methods of our geth fork, not really of "reactive network" 

I also think it's worth mentioning that this methods are required because reactive network consists of many reactvms bla bla

Reply:
Yes, I'll fix that. But I don't understand why you think the following structure is too much:
Overview
Parameters
cURL
Response